content/docs/04-clusters/06-cluster-management/03-compliance-scan.md

```diff
@@ -37,41 +39,93 @@ The scan summary of total passed and failed tests are displayed while the test i
+
+If a [backup location](/clusters/cluster-management/backup-restore) is provided, the SBOM for each image will be uploaded to your backup location and can subsequently be downloaded with the click of a button or using the Palette API. If a backup location is not provided, Palette will still preserve all of the dependencies and vulnerabilities that were identified, but the raw SBOMs will not be available for download. All the results and metadata pictured in the screenshots below will be included even without a backup location configured, except when using the **github-json** format, in which case a backup location must be provided.
+
+<br />
+
+<WarningBox>
+A Backup Location is <b>mandatory</b> when configuring an SBOM scan using the <b>github-json</b> format.
+</WarningBox>
+
+<br />
+
+#### SBOM scan formats
+* [SPDX](https://github.com/spdx/spdx-spec/blob/v2.2/schemas/spdx-schema.json)
+  * A standardized format for representing Software Bill of Materials that is widely used by organizations and governments. It has been around longer than any other SBOM format.
+* [CycloneDX](https://cyclonedx.org/specification/overview/)
+  * An open-source XML format that provides a standard representation of software components and their metadata.
+* GitHub's [dependency submission format](https://docs.github.com/en/rest/dependency-graph/dependency-submission?apiVersion=2022-11-28)
+  * Can be used to power a [dependency review workflow](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review).
+* Syft JSON
+  * Syft's custom SBOM format. Contains the most complete metadata out of the four options.
+
+#### SBOM scan scopes
+* Cluster
+  * Scan **all** images in your K8s cluster
+* Namespace
+  * Scan all images in a particular **Namespace**
+* Label Selector
+  * Scan all images used by all of the Pods matching a **label selector** within a particular Namespace
+* Pod
+  * Scan all images used by a single **Pod**
+
+## Viewing SBOM scan results
+You can click into a completed scan to view a scan report containing additional detail for every image that was scanned. The context column indicates every unique usage of each image, broken out by container name, namespace, and pod name, as each image may be in use by various containers within a given scope. The vulnerability summary column provides a condensed view of the vulnerability report, which can be viewed in greater detail by clicking on any row in the scan report.
+
+![sbom_results.png](sbom_results.png)
+
+Each image details page within the scan report provides a list of dependencies and vulnerabilities. These tables are condensed highlights of the metadata contained in the SBOM that was generated for a particular image. Each dependency’s version and type is displayed, but additional metadata will be included in the SBOM. Exactly what additional metadata is included will depend on the selected SBOM format.
```

Comment:
```suggestion
Each identified image has its own detailed results page. To review the image report, select the **>** button. Each image's details page within the scan report provides a list of dependencies and vulnerabilities.

The tables contained highlights of the metadata in the SBOM for a particular image. Each dependency's version and type are displayed. The amount of displayed details in the metadata section depends on the selected SBOM format. 
```

Flipped the order and clean up the language.

Reply:
Suggest:
The details page for each image in the scan report provides a list of dependencies and vulnerabilities.

The tables list highlights of the metadata in the SBOM for a particular image. Each dependency's version and type are displayed. The amount of displayed details in the metadata section depends on the selected SBOM format.

Reply:
@karl-cardenas-coding @ritawatson one small issue with the re-wording(s) is that the additional metadata is in the SBOM - not on our UI. Users will only notice it if they download the SBOM and inspect it. Our UI shows the same info (dependencies & vulnerabilities) no matter what.

Reply:
@TylerGillson I see, in that case, it sounds like Rita's suggestion is probably closer to what you're thinking. We could end the paragraph by stating the additional metadata is only available in the downloaded format.